nav2_docking/opennav_docking/src/controller.cpp

```diff
@@ -92,6 +199,10 @@ Controller::dynamicParametersCallback(std::vector<rclcpp::Parameter> parameters)
         v_angular_max_ = parameter.as_double();
       } else if (name == "controller.slowdown_radius") {
         slowdown_radius_ = parameter.as_double();
+      } else if (name == "controller.projection_time") {
+        projection_time_ = parameter.as_double();
+      } else if (name == "controller.simulation_time_step") {
+        simulation_time_step_ = parameter.as_double();
```

Comment:
collision_tolerance?

Reply:
Good point, I think the only reason I didn't add it was because collision detection is not toggleable at runtime, but yes, if it's enabled, the *collision_tolerance* could be set as a dynamic parameter.